appserver/tests/embedded/basic/config/src/test/java/org/glassfish/tests/embedded/basic/config/ConfigTest.java

```diff
@@ -55,9 +56,15 @@ static void bootStrap() throws GlassFishException {
 
     @ParameterizedTest
     @ArgumentsSource(ConfigFileArgumentsProvider.class)
-    void testConfigFile(String configFile) throws Exception {
+    void testConfigFile(Object configFile) throws Exception {
         GlassFishProperties gfp = new GlassFishProperties();
-        gfp.setConfigFileURI(configFile);
+        if (configFile instanceof String) {
+            gfp.setConfigFile(new File((String)configFile));
```

Comment:
Is this branch used?

Reply:
Yes, this is used with the argument "domain.xml" created by ConfigFileArgumentsProvider.

Reply:
:grinning: Now I can see. 
:+1: